posthog/src/main/java/com/posthog/PostHogConfig.kt

```diff
@@ -202,6 +203,45 @@ public open class PostHogConfig(
     private val integrationsList: MutableList<PostHogIntegration> = mutableListOf()
     private val integrationLock = Any()
 
+    /**
+     * Hook that allows to sanitize the event
+     * The hook is called before the event is cached or sent over the wire
+     */
+    private val beforeSendBlockList: MutableList<PostHogBlockEvents> = mutableListOf()
+    private val beforeSendBlockLock = Any()
```

Comment:
```suggestion
    private val beforeSendList: MutableList<PostHogBlockEvents> = mutableListOf()
    private val beforeSendLock = Any()
```

Remove block from naming